### DISCUSSION TITLE
Can it seamlessly switch from functions like printf to trice

### DISCUSSION THREAD

**Discussion by xlbao123** [rokath/trice#441] [General]
Can it seamlessly switch from functions like printf, such as

#Define LOG TRICE

And, can various formats such as %d %f %s be supported in a single format string

**Comment by rokath** [comment]
Please read [Trice Similarities and differences to printf usage](https://github.com/rokath/trice/blob/master/docs/TriceVsPrintfSimilaritiesAndDifferences.md#trice-similarities-and-differences-to-printf-usage) and [User Code Adaption](https://github.com/rokath/trice/blob/master/docs/TriceUserGuide.md#255-user-code-adaption) inside the [Trice User Guide](https://github.com/rokath/trice/blob/master/docs/TriceUserGuide.md#trice-user-guide).
